hveto/html.py

```diff
@@ -90,6 +90,82 @@
 """
 
 
+# -- set up default header plot captions
+
+head_hist = "Histogram of number of triggers in the primary channel before " \
```

Comment:
@alurban, typically such global variables use uppercase names, so that they are easily distinguished from local variables. Can you have a think about making that change? e.g.

```python
HEAD_HIST = "..."
HEAD_ROC = "..."
```

Also, we might want to use `_CAPTION` in the variable name, to give a litte more context, or do away with the individual variables altogether and just use the `HEAD_CAPTIONS` and `ROUND_CAPTIONS` `dict`s.